### DISCUSSION TITLE
Is Kaboomjs.com still going?

### DISCUSSION THREAD

**Discussion by ShiftyKitty** [replit/kaboom#625] [Q&A] [answered]
Is kaboomjs still operational? Was looking to mess around with it but the main site is down.

**Comment by lajbel** [comment, marked as answer]
It really is a temporary thing. Kaboom is in a somewhat paused stage and I think it will continue for a few more weeks, soon we will be back to the big development! I recommend you to join the Discord to be updated with upcoming events and more

Kaboom site is now back!! 🐸

**Reply by ShiftyKitty** [reply]
Fantastic. Thanks for letting me know
